Tools/TransformTool/TransformTool.cs

```diff
@@ -363,6 +363,8 @@ public void ProcessInput(ActionMapInput input, Action<InputControl> consumeContr
 #if UNITY_EDITOR
 			if (m_PivotRotation == PivotRotation.Local) // Manipulator does not rotate when in global mode
 				manipulatorTransform.rotation = Quaternion.Slerp(manipulatorTransform.rotation, m_TargetRotation, kLazyFollowRotate * deltaTime);
+
+			UnityEditor.Undo.RecordObjects(Selection.transforms, "EditorVR Manipulator");
```

Comment:
import UnityEditor, so this doesn't have to be fully qualified.

Instead of "EditorVR Manipulator", I'd suggest using our current Unity naming "Move" to match with what happens with the Transform tool in scene view.